packages/integration-objects/srv/exceptions/validation-error.js

```diff
@@ -0,0 +1,10 @@
+/* Copyright (c) 2021 SAP SE or an SAP affiliate company. All rights reserved. */
+class ValidationError extends Error {
+  constructor(message, code) {
+    super(message);
+    this.name = 'ValidationError';
+    this.code = code;
```

Comment:
The error code here seems like a separate concern that should not be mixed with a `ValidationError`. It seems to be an HTTP status code, which is a totally different level of abstraction. However, let's not refactor it on this PR.